packages/manager/src/services/webProperty/types.ts

```diff
@@ -1,17 +1,23 @@
 export type TWebProperty = {
-  _id: string;
+  env: [
```

Comment:
Change syntax to `{}[]` instead of `[{}]`. This is saying there is only single element in a array rather than an array of object